www/pfSense-pkg-squid/files/usr/local/pkg/squid.xml

```diff
@@ -131,6 +131,19 @@
 			<type>checkbox</type>
 			<default_value>on</default_value>
 		</field>
+		<field>
+			<fielddescr>Listen IP Version</fielddescr>
+			<fieldname>listenproto</fieldname>
+			<description>Select listen IP version. This controls what addresses Squid listens for client connections.</description>
```

Comment:
Reword this to `Select the IP version Squid will use to select addresses for accepting client connections.`

Reply:
fixed